implementations/rust/ockam/ockam_examples/example_projects/tcp/Cargo.lock

```diff
@@ -0,0 +1,1458 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Not sure why this lock file appeared here. Could you please remove it?

Reply:
Oops, so many files :laughing: I'll remove it